### PR TITLE
Support more test filename conventions

### PR DESCRIPTION
**Pull request by kentcdodds**
This PR just updates documentation. I couldn't for the life of me find out where this magic happens. Pointers? Thanks!

### CODE REVIEW

**Comment by jamestalmage** [comment]
Wait for #713 to land. Then check `lib/ava-files`

File: cli.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ var cli = meow([
 	'  ava --init foo.js',
 	'',
 	'Default patterns when no arguments:',
-	'test.js test-*.js test/**/*.js'
+	'test.js test-*.js test/**/*.js __tests__/**/*.js **/*.test.js'
```
Comment (sindresorhus) [inline cli.js]: I thought the convention is that `__tests__` can be anywhere? Hence why it's wrapped in underscores.

Comment (kentcdodds) [inline cli.js]: Oh, is my glob wrong? Should it be `**/__tests__/**`? That's what I did for my PR to `nyc`. Forgive my ignorance with `glob` :sweat_smile:

Comment (jamestalmage) [inline cli.js]: :+1: `**/__tests__/**`

Comment (kentcdodds) [inline cli.js]: Updated. :+1: I've subscribed to #713. As soon as that gets merged, I'll give this another look.

Comment (jfmengels) [inline cli.js]: Why not `__tests__/**/*.js`?

**Comment by jfmengels** [comment]
`**/*.test.js` ! I use this all the time, would love it if it landed in the default files :heart: 
We'll need to update the [linter](https://github.com/sindresorhus/eslint-plugin-ava/blob/master/docs/rules/no-ignored-test-files.md) too.

**Comment by kentcdodds** [comment]
This has been updated :+1:

File: lib/ava-files.js [hunk 2]
```diff
@@ -17,7 +17,9 @@ function defaultIncludePatterns() {
 	return [
 		'test.js',
 		'test-*.js',
-		'test'
+		'test',
+		'**/__tests__/**',
```
Comment (novemberborn) [inline lib/ava-files.js]: Are all `**/*.js` files inside `__tests__` tests? If so I think you just need to match `**/__tests__` and AVA's built in directory recursion will take over.

**Comment by kentcdodds** [comment]
I imagine that we'll want to add a test for this, but I can't seem to find existing tests for the default inclusion glob...

File: lib/ava-files.js (hunk 2 above)
Comment (kentcdodds) [inline lib/ava-files.js]: :+1:

Comment (kentcdodds) [inline lib/ava-files.js]: updated

**Comment by novemberborn** [comment]
> I imagine that we'll want to add a test for this, but I can't seem to find existing tests for the default inclusion glob...

Yea. I'm alluding to that in <https://github.com/sindresorhus/ava/issues/736>.

File: cli.js [hunk 3]
```diff
@@ -80,7 +80,7 @@ var cli = meow([
 	'  ava --init foo.js',
 	'',
 	'Default patterns when no arguments:',
-	'test.js test-*.js test/**/*.js'
+	'test.js test-*.js test/**/*.js **/__tests__ **/*.test.js'
```
Comment (novemberborn) [inline cli.js]: Documentation wise it should be `**/__tests__/**/*.js`, just like the `test` directory in `ava-files` is explained as `test/**/*.js`.

Comment (kentcdodds) [inline cli.js]: Good point. Updated :+1:

**Comment by novemberborn** [comment]
LGTM!

**Comment by jfmengels** [comment]
Woohoo! This makes my day, thanks @kentcdodds!

**Pull request merged**